Workiva/w-mobile-kit
# MOBILE-1685: w-mobile-kit: Switch

## Description

Need a custom switch (like webskins). See attached [specs.](https://drive.google.com/a/webfilings.com/file/d/0B_bu90wZe-wyMmJiNkJMN1FpOHM/view?usp=sharing)
## What Was Changed

Added a WSwitch component to w-mobile-kit
## Testing
- Ensure unit tests pass and code coverage is sufficient
- Run example app and navigate to WSwitch and WRadio Examples VC
- Ensure switches match specs, function properly, look good, etc...
- Review how switches are added via storyboard and via code and ensure its explanatory

---

Please Review: @Workiva/mobile
